genesis/process/genesisBlockCreator.go

```diff
@@ -111,6 +109,21 @@ func (gbc *genesisBlockCreator) createHardForkImportHandler() error {
 	return nil
 }
 
+func createStorer(storageConfig config.StorageConfig, folder string) (storage.Storer, error) {
+	dbConfig := factory.GetDBFromConfig(storageConfig.DB)
+	dbConfig.FilePath = path.Join(folder, storageConfig.DB.FilePath)
+	accountsTrieStorage, err := storageUnit.NewStorageUnitFromConf(
```

Comment:
this is not necessarily an accountsTrie storage - only storage

Reply:
done